py/circuitpy_mpconfig.mk

```diff
@@ -207,6 +213,11 @@ CIRCUITPY_ROTARYIO = 1
 endif
 CFLAGS += -DCIRCUITPY_ROTARYIO=$(CIRCUITPY_ROTARYIO)
 
+ifndef CIRCUITPY_COUNTIO
+CIRCUITPY_COUNTIO = 1
+endif
+CFLAGS += -DCIRCUITPY_COUNTIO=$(CIRCUITPY_COUNTIO)
+
```

Comment:
```suggestion
```

Reply:
As of #2799, simple `ifndefs` now use `?=`, e.g.
https://github.com/adafruit/circuitpython/blob/c311b5c002d004d7a985ecdaf8e0344b35dca209/py/circuitpy_mpconfig.mk#L36-L37

Reply:
No problem, thank you. I've changed this.